src/components/AddBook.js

```diff
@@ -13,8 +13,8 @@ export default function AddBook() {
     const newBook = {
       item_id: uuidv4(),
       title: bookTitle,
-      category: 'Random',
-      author: 'Jeff',
+      category: 'Business',
+      author: 'Lewis Hamilton',
```

Comment:
## [OPTIONAL]

- [ ] I would really suggest you implement the categories selection feature and not take the default value `business`

Reply:
@andersonlebon 

Thanks. It is not a requirement, I also left it like that because there is no provision for a user to add a category, so what much be rendered?